### PR TITLE
make possible to use http basic auth with guzzle http client

### PR DESCRIPTION
**Pull request by xeroxzone**
with credentials from doctrine dbal connnection parameters user and password
@see: http://guzzle.readthedocs.org/en/latest/clients.html
@see: http://doctrine-dbal.readthedocs.org/en/latest/reference/configuration.html

### CODE REVIEW

**Comment by Ocramius** [comment]
Tests?

**Comment by xeroxzone** [comment]
Thanks – test added:

on my local machine it runs fine; is there something special in note of scrutinizer.

From my local machine:
vendor/bin/phpunit
PHPUnit 4.6.6 by Sebastian Bergmann and contributors.

Configuration read from /srv/www/symfony/crate-pdo/phpunit.xml.dist

...............................................................  63 / 127 ( 49%)
............................................................... 126 / 127 ( 99%)
.

Time: 6.72 seconds, Memory: 9.50Mb

OK (127 tests, 219 assertions)

File: test/CrateTest/PDO/PDOTest.php [hunk 1]
```diff
@@ -96,6 +96,17 @@ public function testInstantiationWithInvalidOptions()
...
 
     /**
+     * @covers ::__construct
+     */
+    public function testInstantiationWithHttpBasicAuth()
+    {
+        $pdo = new PDO('crate:localhost:1234', 'user', 'passwd', []);
+
+        $this->assertInstanceOf('Crate\PDO\PDO', $pdo);
+        $this->assertInstanceOf('PDO', $pdo);
```
Comment (Ocramius) [inline test/CrateTest/PDO/PDOTest.php]: You need some sort of assertions on the http client parameters

**Comment by xeroxzone** [comment]
I don't see it. Is it me or what this message from Scrutinizer mean: The code coverage data was not received within the specified time. Please make sure your third-party service is configured to send code coverage data.

**Pull request closed without merging**